neuralpredictors/data/datasets.py

```diff
@@ -706,17 +680,18 @@ def add_neuron_meta(self, name, animal_id, session, scan_idx, unit_id, values, f
             values:     new meta information. First dimension must refer to neurons.
             fill_missing:   fill the values of the new attribute with NaN if not provided
         """
+        with HiddenPrints():
+            temp = NewFileTreeDataset(self.dirname, "responses")
```

Comment:
I would do `NewFileTreeDataset(self.dirname, *self.data_keys)`.

Reply:
Per discussion, we suggested that a new context manager is implemented to temporally turn off the transforms

Reply:
Thanks for the suggestion. It's implemented.